Makefile

```diff
@@ -290,11 +290,7 @@ build-and-deploy-ui:
 	&& if [ "$${DEPLOY_UI}" != "false" ]; then ${MAKEFILE_DIR}/devops/scripts/build_deploy_ui.sh; else echo "UI Deploy skipped as DEPLOY_UI is false"; fi \
 
 prepare-for-e2e:
-	$(call workspace_bundle,base) \
-	&& $(call workspace_service_bundle,guacamole) \
-	&& $(call shared_service_bundle,gitea) \
-	&& $(call user_resource_bundle,guacamole,guacamole-azure-windowsvm) \
-	&& $(call user_resource_bundle,guacamole,guacamole-azure-linuxvm)
+	$(call workspace_bundle,airlock_manager)
```

Comment:
I'm guessing this change shouldn't have been committed?

Reply:
Ah, I thought I'd reverted that but my push failed... thanks!